context-free.md

```diff
@@ -1,22 +1,26 @@
 # Context Free
 
 ##  Who is this programming language for?
+Context Free seems to target programmers who are also interested in creating art (computer-generated art is a pretty interesting, though unfortunately small, field).
```

Comment:
Do the features of the language change the intended art to be created, or the intended user?